AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts

```diff
@@ -161,4 +170,14 @@ export class VhoWorkHoursTableComponent implements CanDeactiveComponent {
     get checkVhoHasWorkHours(): boolean {
         return this.workHours?.length > 0;
     }
+    showMessage(message: string) {
+        this.displayMessage = true;
+        this.message = message;
+    }
+
+    checkVhoHasWorkHoursToEdit() {
+        if (this.workHours?.length <= 0) {
```

Comment:
think we'd want to show this message when workHours is null, too

Reply:
? do the null checks